tests/containers/podman_network.pm

```diff
@@ -0,0 +1,52 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: podman network
+# Summary: Test podman network 
+# Maintainer: qac team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use containers::utils qw(registry_url container_ip);
+
+sub run(){
+
+    my ($self, $args) = @_;
+    select_serial_terminal;
+    my $podman = $self->containers_factory('podman');
+
+    record_info('Test', 'Create a new networks named newnet and newnet2');
+    assert_script_run('podman network create newnet');
+    assert_script_run('podman network create newnet2');
+
+    record_info('Test', 'List the created networks');
+    assert_script_run('podman network ls | grep -e newnet -e newnet2');
+
+    record_info('Prep', 'Create two more networks named newnet3 and newnet4');
+    assert_script_run('podman network create newnet3');
+    assert_script_run('podman network create newnet4');
+
+    record_info('Test', 'Delete newnet3 and check list the networks a see if it is deleted');
+    assert_script_run('podman network rm newnet3');
+    assert_script_run('podman network ls | grep -v newnet3');
+
+    record_info('Test', 'Check if newnet4 exists by its ID');
+    my $network_id = assert_script_run('podman network inspect -f {{.Id}} newnet4');
+    assert_script_run('podman network exists $network_id');
+    
+    record_info('Test', 'Inspect the newnet');
+    assert_script_run('podman network inspect newnet --format "{{range .Subnets}}Subnet: {{.Subnet}} Gateway: {{.Gateway}}{{end}}" | grep -e Subnet -e Gateway');
+
+    record_info('Test', 'Remove all unused networks');
+    assert_script_run('podman network prune -f | grep newnet*');
+
+    assert_script_run('podman network ls')
```

Comment:
```suggestion
    assert_script_run('podman network ls');
```

Reply:
Also, what are you actually trying to test with this line?

Reply:
I think that prune action is doing what supposed to do. For me it is fine to have it here, but checking that effectively the list doesn't contains elements
`
assert_script_run('podman network ls | grep -v newnet')
`

Reply:
done